apps/microsoft-teams/frontend/src/components/config/ChannelSelectionModal/ChannelSelectionModal.styles.ts

```diff
@@ -3,8 +3,12 @@ import tokens from '@contentful/f36-tokens';
 
 export const styles = {
   table: css({
-    borderRadius: 'none !important',
+    borderRadius: '0 !important',
     boxShadow: 'none !important',
     borderBottom: `1px solid ${tokens.gray200}`,
   }),
+
+  tableRow: css({
```

Comment:
Good call for adding this! We should also ensure that the cursor is a pointer for the `ContentTypeSelectionModal`, since I don't see the styles file for that component updated.

Reply:
Good call, I have updated `ContentTypeSelectionModal.styles.ts` and added the `classNames={styles.tableRow}` to `ContentTypeSelectionModal.tsx`